static/js/jira-select-card-option.js

```diff
@@ -65,9 +65,10 @@ $(document).ready(function() {
 			);
 		} else {
 			AP.context.getToken(function(token) {
-				const child = openChildWindow("/session?ghRedirect=to&autoApp=1&baseUrl=" + gitHubServerBaseUrl);
+				const child = openChildWindow("/github-manifest");
 				child.window.jiraHost = jiraHost;
 				child.window.jwt = token;
+				child.window.gheHost = gitHubServerBaseUrl;
```

Comment:
I think, `gheHost` is being fetched from `texfield` and can be pointed to mock GHE server URL. Won't it give same VULN? How will you make sure it is legitimate URL?

Reply:
Yup, that can be another VULN. One idea is to store this `gheHost` temporarily in redis or someplace else, but that will be done in another [ticket.](https://softwareteams.atlassian.net/browse/ARC-2035)
These changes are to fix the redirecting VULN for the moment.

Reply:
the original vuln was an open endpoint which could redirect.  This particular endpoint is authenticated behind a JWT token (not open) making it essentially impossible to hijack